.travis/before_script.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ev
+
+# Download Firefox Nightly which is compatible with the Marionette client version used 
+mozdownload -t daily
+
+if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+    hdiutil attach -nobrowse -noautoopen *firefox*.mac.dmg
+    cp -R /Volumes/Nightly/FirefoxNightly.app /Applications
+    cd ~
+    hdiutil unmount /Volumes/Nightly/
+else
+    tar -xvf *firefox*.tar.bz2
+fi
```

Comment:
s already said in my last review I do not have any problems with mozinstall. So we should use that, set the BINARY environment variable via "BINARY=`mozinstall *firefox*`. Then this file is not necessary anymore. Please test this again and if failures occur let me know so I can have a look at. If there is really a blocker we can move this out to a follow-up bug.